openstackgit-update.yml

```diff
@@ -33,22 +33,13 @@
         path: "{{ working_dir }}/openstackgit"
         state: directory
 
```

Comment:
why did you remove the git clone? Is this playbook supposed to be included from another playbook?